langchain4j/src/main/java/dev/langchain4j/service/output/EnumListOutputParser.java

```diff
@@ -15,6 +18,11 @@ class EnumListOutputParser extends EnumCollectionOutputParser<Enum> {
     @Override
     public List<Enum> parse(String text) {
         List<String> stringsList = asList(text.split("\n"));
-        return stringsList.stream().map(enumOutputParser::parse).collect(Collectors.toList());
+        if (text.startsWith("{")) {
+            stringsList = (List<String>) Json.fromJson(text, Map.class).values().stream().findFirst().get();
```

Comment:
I guess it would be more resilient to first try to find "items" and only then fallback to `findFirst()`.
Could you please unit-test this logic?